mod/admin.php

```diff
@@ -762,6 +766,39 @@ function admin_page_queue(App $a)
 	]);
 }
 
+/**
+ * @brief Admin Inspect Worker Queue Page
+ *
+ * Generates a page for the admin to have a look into the current queue of
+ * worker jobs. Shown are the parameters for the job and its priority.
+ *
+ * The returned string holds the content of the page.
+ *
+ * @param App $a
+ * @return string
+ */
+function admin_page_workerqueue(App $a)
+{
+	// get content from the queue table
+	$r = q("SELECT `q`.`id`,`q`.`parameter`, `q`.`created`, `q`.`priority`
+			FROM `workerqueue` AS `q`
+                        WHERE (`q`.`done` = 0)
```

Comment:
There is a additional space in the indentation here.

Reply:
It's okay, the plain SQL has to go anyway. 😉